objects.go

```diff
@@ -198,3 +198,37 @@ func (v *EADefListValue) UnmarshalJSON(b []byte) (err error) {
 	*v = EADefListValue(m["value"])
 	return
 }
+
+type RequestBody struct{
+	Data map[string]interface{} `json:"data,omitempty"`
+	Args map[string]string `json:"args,omitempty"`
+	Method string `json:"method"`
+	Object string `json:"object,omitempty"`
+	EnableSubstitution bool `json:"enable_substitution,omitempty"`
+	AssignState map[string]string `json:"assign_state,omitempty"`
+	Discard bool `json:"discard,omitempty"`
+}
```

Comment:
Is there some restriction on the field values (or combination thereof) based on the request type (i.e. Method="GET" as oppose to Method="PUT")? If so, there should be a factory method for each type of request, e.g. NewGetRequestBody(), with a specific set of arguments..